src/overlays/actors/ovl_En_Diving_Game/z_en_diving_game.c

```diff
@@ -547,22 +547,22 @@ Gfx* EnDivingGame_EmptyDList(GraphicsContext* gfxCtx) {
     return displayList;
 }
 
-s32 EnDivingGame_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx) {
+s32 EnDivingGame_OverrideLimbDraw(PlayState* play, s32 limb, Gfx** dList, Vec3f* pos, Vec3s* rot, void* thisx) {
```

Comment:
I dont necessarily disagree with calling it `limb` instead of `limbIndex`, but this is something that should be changed everywhere at the same time. It is a very common naming convention currently with the skelanime callbacks. I recommend reverting and making a separate PR for this change

Reply:
you're right, better not to indeed. reverted the changes with 41b6509fec2d9f8d3cf6f04bd959f4680c2c6837